recipes/vamb/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "3.0.2" %}
+{% set version = "v3.0.3" %}
 
 package:
   name: vamb
   version: {{ version }}
 
 source:
   url: https://github.com/RasmussenLab/vamb/archive/{{ version }}.tar.gz
```

Comment:
Put the `v` here rather than in the `set version` line.

```suggestion
  url: https://github.com/RasmussenLab/vamb/archive/v{{ version }}.tar.gz
```

Reply:
It seems the developer are not consistent with the version tag. I find it more visible at the top. What are your reasons?

Reply:
`v` isn't actually part of the version and keeping it there can screw up version sorting.